src/js/leaflet.storage.core.js

```diff
@@ -377,11 +378,13 @@ L.Storage.Help = L.Class.extend({
         L.DomUtil.add('h3', '', container,'GeojSON');
         L.DomUtil.add('p', '', container, L._('All properties are imported.'));
         L.DomUtil.add('h3', '', container,'GPX');
-        L.DomUtil.add('p', '', container, L._('Properties imported:') + 'name, desc');
+        L.DomUtil.add('p', '', container, L._('Properties imported: ') + 'name, desc');
```

Comment:
If you add the space inside the translated string, you break all the translations ;)